wordpress-mobile/gutenberg-mobile
# Improve media placeholders during upload

Fixes #1544 
### Companion PR's
Gutenberg PR: https://github.com/WordPress/gutenberg/pull/19497

### Changes
- This change is introducing deliberate, consistent styles for both light and dark mode for image and video placeholder icons and backgrounds during upload.
- There is also some small refactoring for fetching icons in an easier to comprehend way and removing unreachable code.

### To Test:
Uploading images can be tested using the WordPress Android and iOS apps or the Example App using the following steps:

1. Add new Image or Video block
2. Click to upload a new image from wordpress media library or from device
3. Note that during upload we briefly see the improved placeholder.

### Screenshots

Before:
![image](https://user-images.githubusercontent.com/1103838/71933964-32f4ab80-3158-11ea-9dcd-16be505cd1ef.png)

After:
![image](https://user-images.githubusercontent.com/1103838/71933973-3d16aa00-3158-11ea-945f-52c805b1e0a4.png)

Video:
![image-upload-gif](https://user-images.githubusercontent.com/1103838/71934678-d09caa80-3159-11ea-8c23-1d58473abb28.gif)


PR submission checklist:
